packages/odyssey-react-mui/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { ReactElement } from "react";
+
+import { odysseyTheme } from "../../themes";
+
+export const ThemeProvider = ({ children }: { children: ReactElement }) => (
```

Comment:
will this component will accept theme as an attribute ? 
If not Can we change the name to OdysseyThemeProvider

Reply:
You can import `MuiThemeProvider` instead of `ThemeProvider` if you want to specify the theme yourself.

Most apps won't need this, so having a wrapped `ThemeProvider` removes unnecessary boilerplate.